web/templates/user_profile_modal.hbs

```diff
@@ -104,6 +104,18 @@
                     </div>
 
                     <div class="tabcontent" id="user-profile-groups-tab">
+                        {{#unless is_guest}}
+                        <div class="manage_user_group">
+                            {{#if can_edit_user_groups}}
+                            {{t "View and edit user groups" }}
+                            {{else}}
+                            {{t "View user groups" }}
+                            {{/if}}
+                            <a href="#organization/user-groups-admin">
+                                here.
+                            </a>
+                        </div>
+                        {{/unless}}
```

Comment:
Translations don't work this way. See 
https://zulip.readthedocs.io/en/latest/translating/internationalization.html and especially the parts about word order varying with language.

Reply:
> Translations don't work this way. See https://zulip.readthedocs.io/en/latest/translating/internationalization.html and especially the parts about word order varying with language.

@timabbott I've pushed the new changes. Could you please check if they're correct?

I Referred to the following block of code from [here](https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations).

```
{{#tr}}
    <b>HTML</b> linking to the <z-link>login page</z-link>
    {{#*inline "z-link"}}<a href="/login/">{{> @partial-block}}</a>{{/inline}}
{{/tr}}
```

Reply:
I checked with @alya and confirmed that in our current writing patterns, we should just have the whole phrase "View and edit user groups" or "View user groups" be a link, with no "here"; that probably also means you no longer need the more complex i18n construction.

Reply:
> I checked with @alya and confirmed that in our current writing patterns, we should just have the whole phrase "View and edit user groups" or "View user groups" be a link, with no "here"; that probably also means you no longer need the more complex i18n construction.

@timabbott I've pushed the code after making these changes.

Reply:
Please update the screenshots in the PR description accordingly.

Reply:
> Please update the screenshots in the PR description accordingly.

Sorry for the delay, I've updated the screenshots and screen captures.

Reply:
I still see a "here" in the screenshots.